grpc/cubic.proto

```diff
@@ -235,6 +293,55 @@ message RecognitionMetadata {
   string custom_metadata = 1;
 }
 
+// Data that may provide useful context to assist the speech recognition. This
+// can be used to improve the transcription of particular phrases or words such
+// as proper names or commands for example. This can also be used to add out of
+// vocabulary words to the model. We currently support pre-compiled context
+// information  via the `CompileContext` method, generated from lists of words
+// or phrases expected to appear in the recognition output.
```

Comment:
```suggestion
  // A collection of additional context information that may aid
  // speech recognition.  This can be used to add out-of-vocabulary words to  
  // the model or to boost recognition of specific proper names or commands. 
```